app/models/profile_link.rb

```diff
@@ -26,4 +26,11 @@ class ProfileLink < ApplicationRecord
   belongs_to :profile_link_site, required: true
 
   validates :url, presence: true
+
+  def url=(value)
+    self.attributes[:url] = value.gsub(
```

Comment:
nope same error: `* profile_link - Validation failed: Url can't be blank (ActiveRecord::RecordInvalid)`.